sql/plan/create_index.go

```diff
@@ -110,6 +110,10 @@ func (c *CreateIndex) RowIter(ctx *sql.Context) (sql.RowIter, error) {
 	if err != nil {
 		return nil, err
 	}
+	riter, err := table.RowIter(ctx)
```

Comment:
why are we using the `RowIter` and `IndexKeyValueIter`? `IndexKeyValueIter` should already return everything you need to create an index. If there's something missing, it should go there.

The problem is `colNames` is string instead of `sql.Expression`, right? Perhaps then the solution should be to change the `IndexKeyValueIter` signature to have that.

Reply:
yeah, kind of for indexes like *LANG(col1, col2)* `IndexKeyValueIter` returns column names instead of evaluate language. Here I used 2 iterators just like a quick fix/workaround, because  `IndexKeyValueIter`  gives me *location* and with `RowIter` I can `Eval` expression.

Reply:
But those iterators are not guaranteed to return the rows in the same order, so that's kind of dangerous. I mean, ours might (i'm not really sure), but if someone implements their own most probably not.